definitions/json/ComplexTypes.json

```diff
@@ -5320,6 +5320,77 @@
     "ElementLabel": "Short Description",
     "SecurityClassification": "Public"
   },
+  {
+    "ID": "replyDocumentCollection",
+    "ListElementCode": "uploadedDocument",
+    "FieldType": "Document",
+    "ElementLabel": "Document",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "replyDocumentCollection",
+    "ListElementCode": "shortDescription",
+    "FieldType": "Text",
+    "ElementLabel": "Short Description",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "referralReply",
+    "ListElementCode": "replyBy",
+    "FieldType": "Text",
+    "ElementLabel": "Reply by",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "referralReply",
+    "ListElementCode": "directionTo",
+    "FieldType": "Text",
+    "ElementLabel": "Reply to",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "referralReply",
+    "ListElementCode": "replyToEmailAddress",
+    "FieldType": "Text",
+    "ElementLabel": "Email address",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "referralReply",
+    "ListElementCode": "replyDate",
+    "FieldType": "Text",
+    "ElementLabel": "Date",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "referralReply",
+    "ListElementCode": "isUrgentReply",
+    "FieldType": "Text",
+    "ElementLabel": "Urgent",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "referralReply",
+    "ListElementCode": "directionDetails",
+    "FieldType": "Text",
+    "ElementLabel": "Directions",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "referralReply",
+    "ListElementCode": "replyDocument",
+    "FieldType": "Collection",
+    "ElementLabel": "Documents",
+    "FieldTypeParameter": "replyDocumentCollection",
```

Comment:
Thinking this could also be potentially replaced with DocumentUpload?

Reply:
Yep, replaced.